src/Microsoft.Identity.Web/TokenCacheProviders/TokenCacheExtensions.cs

```diff
@@ -2,6 +2,8 @@
 // Licensed under the MIT License.
 
 using System;
+using System.Collections.Generic;
```

Comment:
There are no unit tests

Reply:
I know for the UTs. I just wanted to have a branch for customers to fix